src/devices/vdml_motors.cpp

```diff
@@ -205,19 +205,31 @@ motor_pid_full_s_t Motor::convert_pid_full(double kf, double kp, double ki, doub
 }
 
 std::int32_t Motor::set_pos_pid(const motor_pid_s_t pid) const {
+	#pragma GCC diagnostic push
```

Comment:
Alternatively, you could do this all in one pragma/pop

Reply:
ah that's a good point